src/index.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+import fullCorvidTypes from "../dist/fullCorvidTypes.json";
+import wixModulesNames from "../dist/wixModules.json";
+import eventHandlersService from "./dynamicTypes/eventHandlersService";
+import { TS_CONFIG_PATHS } from "./constants";
+import dynamicTypings from "./dynamicTypes";
+
+export * from "./dynamicTypes/eventHandlersService";
+
+module.exports = {
```

Comment:
kinda hard to see what has changed (`git mv`!)
why we mix es and cjs imports? es is the way to go IMO

Reply:
I mix es and cjs imports because a lot of tests still use require and also other libraries that dynamically import corvid-types.